src/components/NFTPortfolio/Card.vue

```diff
@@ -19,3 +19,24 @@
     <slot />
   </div>
 </template>
+
+<script>
+import { NFT_GEM_NAME } from '~/constant';
+
+export default {
+  props: {
+    gemLevel: {
+      type: Number,
+      default: -1,
+    },
+  },
+  computed: {
+    hoverClass() {
+      if (this.gemLevel < 0 || this.gemLevel > NFT_GEM_NAME.length) {
+        return 'border-like-cyan-light';
+      }
+      return `border-gem-${NFT_GEM_NAME[this.gemLevel].toLowerCase()}`;
```

Comment:
I think it is more simple to just return `border-[#xxxxxx]` for each gem level without changing `tailwind.config.js`

Reply:
tailwind won't define the css for these classes if I dynamically return it in function though, since it can't see them in source code. Or is there any trick to make them work even in dynamic cases?

Reply:
Don't construct class names dynamically
Ref: https://tailwindcss.com/docs/content-configuration#dynamic-class-names

I think just returning the complete class name is more simple
```js
const gemKey = NFT_GEM_NAME[this.gemLevel].toLowerCase();
switch (gemKey) {
  case 'dust':
    return 'hover:border-[#f7f7f7]';

  case 'stone':
  case 'rock':
  case 'boulder':
    return 'hover:border-[#ebebeb]';
  ...
}
```

Reply:
I think it is more elegant to place `hoverClass` prop inside `<NFTGemWrapper>` and return it through slot props and set it to the class of `<NFTPortfolioCard/>` inside `<NFTPortfolioBase/>`